fisher.fish

```diff
@@ -16,18 +16,42 @@ end
 function $fisher_cmd_name -d "fish plugin manager"
     switch "$FISH_VERSION"
         case 2.1.2 2.1.1 2.1.0 2.0.0
-            __fisher_log error "You need fish &2.2.0& or higher to use fisherman."
+            echo "You need fish 2.2.0 or higher to use fisherman."
 
-            if command -s brew > /dev/null
-                __fisher_log info "Run &brew up; brew upgrade --HEAD fish&"
+            if type brew >/dev/null 2>&1
```

Comment:
`command -s` was only introduced in `fish 2.2.0` (a version we are sure the user doesn't have installed) . Without `-s` it actually executes the command even with `>/dev/null 2>&1`. `type` doesn't